example/src/components/ArrowBoxExamples.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import ArrowBox from '../../../src/components/ArrowBox';
+import Code from '../../../src/components/Code';
+import Colors from '../../../src/components/Colors';
```

Comment:
This should be shown in the example code (or else we should just use hex values)